docker-compose.prod.yml

```diff
@@ -41,40 +41,31 @@ services:
       db:
         condition: service_healthy
     volumes:
-      - ./thesis_uploads:/thesis_application_uploads
-      - ./mailing_templates:/mailing_templates
+      - ./thesis_uploads:/uploads
     expose:
       - "8080"
     environment:
       - TZ=Europe/Berlin
-      - ENVIRONMENT
-      - SPRING_DATASOURCE_URL
+      - SPRING_JPA_HIBERNATE_DDL_AUTO=update
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://db:5432/${SPRING_DATASOURCE_DATABASE}
```

Comment:
The database initialization is PostgreSQL code that is not fully compatible with MySQL so this would require larger changes which I would avoid for now